db/migrate/20160226103946_add_location_to_shop.rb

```diff
@@ -0,0 +1,6 @@
+class AddLocationToShop < ActiveRecord::Migration
+  def change
+    add_column :shops, :longitude, :numeric
```

Comment:
Does numeric accept decimal points and negative numbers? Don't we need decimal to save decimal points in the coordinates? Also index the two columns for faster queries. 

`
# long, lat coordinates
    add_column :shops, : longitude, :decimal
    add_column :shops, : latitude, :decimal

    # index for faster search
    add_index  :shops, [:lat, :lng]
`